src/indicators/drought/ecmwf_seasonal.py

```diff
@@ -0,0 +1,301 @@
+"""
+Download raster data from ECMWF's seasonal forecast for selected areas
+and combines all dates into one dataframe
+"""
+from pathlib import Path
+import logging
+import time
+import os
+from typing import List
+import numpy as np
+import xarray as xr
+import cdsapi
+
+from src.indicators.flooding.glofas.area import Area
+
+
+DATA_DIR = Path(os.environ["AA_DATA_DIR"])
+RAW_DATA_DIR = DATA_DIR / "raw"
+PROCESSED_DATA_DIR = DATA_DIR / "processed"
+ECMWF_SEASONAL_DIR = Path("ecmwf_seasonal_data")
+CDSAPI_CLIENT = cdsapi.Client()
+DEFAULT_VERSION = 5
+
+logger = logging.getLogger(__name__)
+
+class EcmwfSeasonal:
```

Comment:
There is a lot of repetition between this class and the base GloFAS class -- some of the methods are identical. I think you could probably create a more general base class (maybe a CDS class?), which could go in utils_general or something, and then in flooding / drought we could have the respective glofas / ecmwf subclasses

Reply:
Yes agreed! Not having the time for it now, but made an issue #83